queueUI/mainwindow.py

```diff
@@ -32,6 +33,25 @@ def __init__(self):
         settings.restoreWindowGeometry(self, "mainwindow_geometry")
         print("Settings are saved in:", settings.fileName())
 
+        self.RM = REManagerAPI()
+        self.runEngineOpenButton_3.clicked.connect(self.openRunEngine)
+        self.runEngineCloseButton_3.clicked.connect(self.closeRunEngine)
+
+    # RE Function:
+    def openRunEngine(self):
+        self.RM.environment_open()
+        self.RM.wait_for_idle()
+
+        print(f"status={self.RM.status()}")
+
+    def closeRunEngine(self):
+        self.RM.environment_close()
```

Comment:
Add a check to avoid attempting to close when env is already close (related to #15)